tests/ui/test_capacity_breakdown_ui.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier2,
+    skipif_ui_not_support,
+    bugzilla,
+)
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs.ui.validation_ui import ValidationUI
+
+
+logger = logging.getLogger(__name__)
+
+
+class TestCapacityBreakdownUI(ManageTest):
+    """
+    Test Capacity Breakdown UI
+
+    Test Process:
+    1.Create new project
+    2.Create pvc with relevant sc [ceph-rbd,cephfs]
+    3.Create pod
+    4.Run FIO
+    5.Verify new project created on Capacity Breakdown UI
+    6.Verify new pod created on Capacity Breakdown UI
+
+    """
+
+    @pytest.mark.parametrize(
+        argnames=["project_name", "pod_name", "sc_type"],
+        argvalues=[
+            pytest.param(
+                *["rbd1", "rbd2", constants.CEPHBLOCKPOOL_SC],
```

Comment:
It's always recommended to use the helper function to generate random names for projects, pods, etc. No need to hard-code the names and use them in parametrize.

Reply:
when the name of pod/project is long, you cant find it on the DOM
![image](https://user-images.githubusercontent.com/61982127/135269421-2a589d4c-65a0-4610-a81f-9f898768041e.png)